windows-2019-agent.json

```diff
@@ -0,0 +1,51 @@
+{
+  "variables": {
+    "maven_version": "3.6.3",
+    "git_version": "2.25.1",
+    "java_version": "11.0.6+10"
+  },
+  "builders": [
+    {
+      "type": "azure-arm",
+      "vm_size": "Standard_D4_v3",
+      "os_type": "Windows",
+      "image_publisher": "MicrosoftWindowsServer",
+      "image_offer": "WindowsServer",
+      "image_sku": "2019-Datacenter-Core",
+
+      "communicator": "winrm",
+      "winrm_use_ssl": true,
+      "winrm_insecure": true,
+      "winrm_username": "packer",
+      "winrm_timeout": "20m",
+
+      "client_id": "{{user `client_id`}}",
+      "client_secret": "{{user `client_secret`}}",
+      "subscription_id": "{{user `subscription_id`}}",
+
+      "location": "{{user `location`}}",
+      "managed_image_name": "jenkins-agent-win2019-{{isotime \"20060102150405\"}}",
```

Comment:
I know we are doing the same for the ubuntu template but wouldn't it be better to **not** use isotime in the name so we don't always have to update our jenkins configuration (ci, trusted,...). If for some reasons the image is broken, we have to fix it as soon as possible anyway. 
Or we could build two images, one latest and one with a specific tag.
Personnally I am more in favor for the latter

Reply:
@timja any opinions?

Reply:
I would just try removing the isotime from it, and see how it goes, much nicer if we don't have to update it the other option is a shared image gallery, we could automatically add it to the gallery here and update the latest tag, but that would still allow downgrading if required

Reply:
```suggestion
      "managed_image_name": "jenkins-agent-win2019",
```